pkg/loader/load.go

```diff
@@ -739,8 +739,8 @@ func (b *batchManager) execAccumulatedDMLs() (err error) {
 	if b.fDMLsSuccessCallback != nil {
 		b.fDMLsSuccessCallback(b.txns...)
 	}
-	b.txns = b.txns[:0]
-	b.dmls = b.dmls[:0]
+	b.txns = nil
```

Comment:
maybe we should assign nil to each element to release reference before `b.txns = b.txns[:0]`

Reply:
https://stackoverflow.com/questions/50732829/will-gc-collect-the-objects-while-the-array-set-to-nil-in-golang